packages/ui-extensions-react/src/surfaces/checkout/components/PaymentIcon/PaymentIcon.ts

```diff
@@ -2,7 +2,6 @@ import {PaymentIcon as BasePaymentIcon} from '@shopify/ui-extensions/checkout';
 import {createRemoteReactComponent} from '@remote-ui/react';
 import type {ReactPropsFromRemoteComponentType} from '@remote-ui/react';
 
-export type {PaymentMethod} from '@shopify/ui-extensions/checkout';
```

Comment:
Remove change from release. This will be added in `2025-07-RC` right after the stable `2025-04`.